lib/engine/step/g_1856/escrow.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative '../base'
+
+module Engine
+  module Step
+    module G1856
+      class Escrow < Base
+        # Since *any* corporation could destinated after a given corporation does a tile lay, we need
+        # to check for *all* corporations, and since multiple corporations could destinate at once
+        # we need to be able to support multiple destinating at once
+        def auto_actions(entity)
+          [
+            Engine::Action::Auto.new(
+              entity,
+              details: @game.corporations.select { |c| @game.destinated?(c) }.compact.map(&:id).join('/')
```

Comment:
Assuming it works, this would be better as a list

Reply:
It's a list now